ParserC.y

```diff
@@ -505,8 +505,8 @@ class Commented c where
 instance Commented Identity where
   withComments = const
 
-instance Commented (State [String]) where
-  withComments (F mf) s = F $ do { f <- mf; put s; return f }
+instance Commented (Writer [String]) where
+  withComments (F mf) s = F $ do { f <- mf; tell s; return f }
```

Comment:
The behavior appears to change from “replace” to “append,” but in reality, there is no difference.

The only place `withComments` is used is 

https://github.com/DanielSchuessler/logic-TPTP/blob/b02af748808c4ad7d120ad4ca2c85ef82a3683a4/ParserC.y#L148-L150

and the `quantifier` is defined as

https://github.com/DanielSchuessler/logic-TPTP/blob/b02af748808c4ad7d120ad4ca2c85ef82a3683a4/ParserC.y#L188-L189

and `for_all` / `exists` are defined as:

https://github.com/DanielSchuessler/logic-TPTP/blob/b02af748808c4ad7d120ad4ca2c85ef82a3683a4/Codec/TPTP/Base.hs#L120-L126

Therefore, `mf` in the above code is always `point` of something, i.e. the annotation is empty.